### PR TITLE
Added tasks 1556, 1557, 1558, 1559, 1560

### PR DESCRIPTION
**Pull request by ThanhNIT**

### CODE REVIEW

**Comment by javadev** [comment]
There is no test for 1556.

image

File: src/test/java/g1501_1600/s1556_thousand_separator/SolutionTest.java [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+import g1501_1600.s1540_can_convert_string_in_k_moves.Solution;
+import org.junit.jupiter.api.Test;
+
+class SolutionTest {
+    @Test
+    void canConvertString() {
+        assertThat(
+                new g1501_1600.s1540_can_convert_string_in_k_moves.Solution()
+                        .canConvertString("input", "ouput", 9),
```
Comment (javadev) [inline src/test/java/g1501_1600/s1556_thousand_separator/SolutionTest.java]: It looks like other task in test.

Comment (ThanhNIT) [reply src/test/java/g1501_1600/s1556_thousand_separator/SolutionTest.java]: let me check

**Comment by javadev** [comment]
➕

**Pull request merged**